test-requirements.txt

```diff
@@ -1 +1,2 @@
 odoo_test_helper
+git+https://github.com/OCA/product-attribute@refs/pull/1512/head#subdirectory=setup/base_product_mass_addition
```

Comment:
No longer needed; https://github.com/OCA/product-attribute/pull/1512 is merged

Reply:
done.